app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -48,6 +48,7 @@ def assign
       target_type = params.require(:target_type)
       username = params.permit(:username)['username']
       group_name = params.permit(:group_name)['group_name']
+      note = params.permit(:note)['note']
```

Comment:
```suggestion
      note = params.permit(:note)['note'].presence
```

Just to avoid having both null and blank notes.

You could even go with:

```suggestion
      note = params.permit(:note)['note'].presence&.strip
```

to clean up the input a bit

Reply:
Alternatively, this could be handled in `Assigner` class 🤷

Reply:
Good point. Keeping it in the controller as I prefer sanitising params there.